content/challenges/easy/3 - Movies/func.js

```diff
@@ -17,5 +17,15 @@
  * @returns {boolean}
  */
 module.exports = (customerMoney, isMovieFull, rating, age, isParentWith) => {
-    // Your code here
-};
+   let saleAllowed = true;
+
+   if (customerMoney < 8) {
+    saleAllowed = false;
+   } if (isMovieFull === true) {
+    saleAllowed = false;
+   } if ((rating === "PG" || rating === "R") && (!isParentWith && age < 17)) {
+    saleAllowed = false;
+   }
+
+   return saleAllowed;
+};
```

Comment:
You'll get a lot of developers with lots of opinions on the creation of the `saleAllowed` variable at all. I tend to find that old school coders try to minimize creation of as many variables as possible while new school don't care because the compilers (minifiers) of our code optimize that for them.

I am of the latter camp. You code is more readable because the variable name signals the function and intent of function entirely. Good job.